AC-Meeting-Guidelines.md

```diff
@@ -0,0 +1,41 @@
+# Architecture Committee Meeting Guidelines
+
+The Architecture Committee (AC) recommends following a set of guidelines for raising issues and adding topics to the meeting agenda.
+
+## Use a GitHub issue or an email to the mailing list first
+
+For a project like Kata Containers, with contributors spanning across so many different time zones, asynchronous discussions and reviews are the preferred communication media.
+
+Directly starting a proposal or trying to resolve new technical issues through the AC meeting has many drawbacks:
+1. It potentially leaves a large part of the community out of the initial dicussion.
+2. It does not give the AC meeting audience enough time to think through the proposal and give constructive and well thought feedback.
+3. It makes it hard to track and share the discussion outcome.
+
+As a consequence, the AC recommends using GitHub issues and PRs, or the [kata-dev mailing list](http://lists.katacontainers.io/cgi-bin/mailman/listinfo/kata-dev) at first before
+adding new items to the AC meetings agenda, especially if you think the topic could be discussed and resolved in less than two weeks.
+
+## AC Agenda topics must be supported by a GitHub issue or PR, or an email sent to the mailing list
+
+Except for a few cases listed in the next section, any topic brought to the AC meeting must be linked to a GitHub issue, PR, or to an email to the mailing list.
+
+Unless the topic must be urgently discussed, the AC recommends waiting for one or two weeks before bringing the discussion to the AC meeting. That will give enough time for interested contributors to digest the issue or proposal before having a live discussion in the AC meeting.
+ 
+
+### Exceptions
+
+* Announcements
+* Presentations
+* Call for reviews of articles, blog posts, etc.
+* [Vulnerabilities disclosures](https://github.com/kata-containers/community/blob/master/VMT)
```

Comment:
Hmm.   I really am not a big fond of adding this here.  VMT is handled by a different group of people than the people who are in the AC.  There may be some overlap, but there may not be some oeverlap.

What worries me about having it here is that it gives people the idea to join the meeting and expose a vulnerability, instead of going through the process of reporting it to a specific group, which will them handle the whole thing, and then report to the community when the vulnerability is fixed.